### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by samuel-rosa**
* Improves documentation of `read_municipality`: adds link to webpage with identification codes for states and municipalities
* Make use of `markdown` in documentation

### CODE REVIEW

File: r-package/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 docs
+geobr.code-workspace
```
Comment (rafapereirabr) [inline r-package/.gitignore:2]: Why is this necessary?

File: r-package/R/read_municipality.R [hunk 2]
```diff
@@ -1,139 +1,139 @@
...
+#' Download shape files of Brazilian municipalities as sf objects
+#'
+#' Data at scale 1:250,000, using Geodetic reference system "SIRGAS2000" and CRS(4674).
+#'
+#' @param year Year of the data (defaults to 2010).
+#' @param code_muni The 7-digit identification code of a municipality. If `code_muni = "all"`
+#' (default), all municipalities of the country will be downloaded. Alternativelly, if the two-digit
+#' identification code or a two-letter uppercase abbreviation of a state is passed, e.g. `33` or
+#' `"RJ"`, all municipalities of that state will be downloaded. Municipality identification codes
+#' are defined in \url{https://www.ibge.gov.br/explica/codigos-dos-municipios.php}.
```
Comment (rafapereirabr) [inline r-package/R/read_municipality.R:10]: The link seems to be broken

File: r-package/.gitignore (hunk 1 above)
Comment (samuel-rosa) [reply r-package/.gitignore:2]: Sorry. My mistake.

File: r-package/R/read_municipality.R (hunk 2 above)
Comment (samuel-rosa) [reply r-package/R/read_municipality.R:10]: This is curious. It was working when I included it. I will try to find a solution ASAP.

Comment (samuel-rosa) [reply r-package/R/read_municipality.R:10]: The website seems to be under maintenance -- see https://cidades.ibge.gov.br/.

File: r-package/.gitignore (hunk 1 above)
Comment (rafapereirabr) [reply r-package/.gitignore:2]: No problem. try to delete this change from the PR

**Comment by samuel-rosa** [comment]
To guarantee that everything is fine, I am running a CRAN check. Found two errors:

```
Error: processing vignette 'intro_to_geobr.Rmd' failed with diagnostics:
   object 'is_R_CMD_check' not found
   --- failed re-building ‘intro_to_geobr.Rmd’
   
   SUMMARY: processing the following file failed:
     ‘intro_to_geobr.Rmd’
   
   Error: Vignette re-building failed.
   Execution halted
```

Found the solution here: https://community.rstudio.com/t/object-is-r-cmd-check-not-found/94995

The second error is as follows:

```
> checking package dependencies ... ERROR
  Package required and available but unsuitable version: ‘ggplot2’
  
  See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
  manual.
```

I am not sure how to solve this error.

**Comment by samuel-rosa** [comment]
OK. Solved it. It seems that I had two versions of the ggplot2 package installed. Not sure how this happened. Anyway, I am not a user of this package.

File: r-package/.gitignore (hunk 1 above)
Comment (samuel-rosa) [reply r-package/.gitignore]: Done

File: r-package/R/read_immediate_region.R [hunk 3]
```diff
@@ -1,68 +1,63 @@
-#' Download shape files of Brazil's Immediate Geographic Areas as sf objects
```
Comment (rafapereirabr) [inline r-package/R/read_immediate_region.R:1]: What exactly has been changed here? Github indicates that you have replaced the whole function

File: r-package/R/read_intermediate_region.R [hunk 4]
```diff
@@ -1,68 +1,63 @@
-#' Download shape files of Brazil's Intermediate Geographic Areas as sf objects.
```
Comment (rafapereirabr) [inline r-package/R/read_intermediate_region.R:1]: What exactly has been changed here? Github indicates that you have replaced the whole function

Comment (samuel-rosa) [reply r-package/R/read_intermediate_region.R:1]: Curious. In all files, I removed the following `library(geobr)` in the examples. In `r-package/R/read_intermediate_region.R`, I also remove the period at the end of the function title. I run a CRAN check and got no errors.

**Review by rafapereirabr — APPROVED** [review]

**Comment by rafapereirabr** [comment]
Thank you for the contribution, Samuel.

**Comment by samuel-rosa** [comment]
I may make more contributions to the documentation in the next months because the package will be used by my undergrad students at UTFPR.
